doc/classes/String.xml

```diff
@@ -640,6 +640,7 @@
 			<description>
 				Splits the string by a [code]delimiter[/code] string and returns an array of the substrings, starting from right.
 				The splits in the returned array are sorted in the same order as the original string, from left to right.
+				If [code]allow_empty[/code] is [code]true[code/], and there are two adjacent delimiters in the string, it will add an empty string to the array of substrings at this position.
```

Comment:
You got the markup to close `[code]` tags the wrong way around, it should be `[/code]`.

Please use `git commit --amend` and `git push --force` to update the commit as described in https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request